Adds Chef Related Plants To Seed Crates

Ahzpy opened the pull request:
<!-- By ticking or leaving ticked the option "Allow edits and access to secrets by maintainers" you give permission for repository maintainers to push changes to your branch without explicitly asking. -->

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do
As an alternative to #30393, the seed crate that can be ordered from cargo now contains some seeds that will be more useful to a chef without botanists.
<!-- Include a small to medium description of what your PR changes. -->
<!-- Document all changes, as not doing this may delay reviews or even discourage maintainers from merging your PR! -->
<!-- If your PR fixes an issue, add "Fixes #1234" somewhere in the PR description. This will automatically close the bug upon PR submission. -->

## Why It's Good For The Game
The Chef needs an _optional_ way to get on their feet in the event there are no botanists to provide basic plants.
<!-- Add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Images of changes

<!-- If you did not make a map or sprite edit, you may delete this section. You may include a gif or mp4 of your feature if you want. -->

## Testing
Spawn crate, has the listed seeds.
<!-- How did you test the PR, if at all? -->

## Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  <!-- Replace the box with [x] to mark as complete. -->
  <!-- Ensure there are no spaces between the x and the square brackets [] else this will not work properly. -->

## Changelog

:cl:
tweak: Seeds Crate now contains seeds that will be more useful to the Chef.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->
<!-- If a PR has no impact on players (i.e. a code refactor that does not change functionality) then the entire Changelog heading and contents can be removed. -->

BiancaWilkson commented:
I like the additions, but is there a specific reason to remove cotton and sunflower seeds from the crate? Cotton can be useful for making clothes/mutating into durathread and I think it would be a good idea to let assistants be able to grow it if they want. Sunflowers are much less useful but can still make for nice headgear.

Ahzpy commented:
> I like the additions, but is there a specific reason to remove cotton and sunflower seeds from the crate? Cotton can be useful for making clothes/mutating into durathread and I think it would be a good idea to let assistants be able to grow it if they want. Sunflowers are much less useful but can still make for nice headgear.
> 
> Edit: Objecting for that reason but I can be convinced

I removed cotton and sunflower seeds as a way of balancing out the crate. I wasn't sure if just adding seeds would affect the price of the crate, so I removed the ones that would be least useful to the Chef or anyone else buying seeds (you hardly see botanists themselves growing or being requested to grow cotton or sunflowers). I can add them back if they don't need to be removed.

Stoniest commented:
> > I like the additions, but is there a specific reason to remove cotton and sunflower seeds from the crate? Cotton can be useful for making clothes/mutating into durathread and I think it would be a good idea to let assistants be able to grow it if they want. Sunflowers are much less useful but can still make for nice headgear.
> > Edit: Objecting for that reason but I can be convinced
> 
> I removed cotton and sunflower seeds as a way of balancing out the crate. I wasn't sure if just adding seeds would affect the price of the crate, so I removed the ones that would be least useful to the Chef or anyone else buying seeds (you hardly see botanists themselves growing or being requested to grow cotton or sunflowers). I can add them back if they don't need to be removed.

Probably best to add them back, it'll be fine at the same cost from my perspective.

PollardTheDragon reviewed (approved):
Code looks good
